simulator/mips/mips_register/mips_register.cpp

```diff
@@ -8,6 +8,7 @@
 const MIPSRegister MIPSRegister::mips_hi = MIPSRegister( MIPS_REG_hi);
 const MIPSRegister MIPSRegister::mips_lo = MIPSRegister( MIPS_REG_lo);
 const MIPSRegister MIPSRegister::mips_hi_lo = MIPSRegister( MIPS_REG_hi_lo);
+const MIPSRegister MIPSRegister::mips_acc = MIPSRegister( MIPS_REG_acc);
```

Comment:
I don't see a reason to have one more MIPS register. We should add data to HI/LO (which is `mips_hi_lo`), right?

Reply:
Where is `MIPS_REG_acc` defined?